released/packages/coq-compcert/coq-compcert.2.7.1/opam

```diff
@@ -0,0 +1,43 @@
+opam-version: "2.0"
+maintainer: "Maxime Dénès <mail@maximedenes.fr>"
+homepage: "http://compcert.inria.fr/"
+dev-repo: "git+https://github.com/AbsInt/CompCert.git"
+bug-reports: "https://github.com/AbsInt/CompCert/issues"
+license: "INRIA Non-Commercial License Agreement"
+build: [
+  [
+    "./configure"
+    "ia32-linux" {os = "linux"}
+    "ia32-macosx" {os = "macos"}
+    "ia32-cygwin" {os = "cygwin"}
+    "-bindir"
+    "%{bin}%"
+    "-libdir"
+    "%{lib}%/compcert"
+    "-clightgen"
+  ]
+  [make "-j%{jobs}%"]
+]
+install: [
+  [make "install"]
+]
+remove: [
```

Comment:
As usual, please delete the `remove` and `flags` lines. I would also appreciate if you could add the required `tags`, with `logpath` and so on.